packages/components/psammead-social-embed/src/utilities/index.js

```diff
@@ -37,15 +34,15 @@ export const dictionaryFactory = ({ provider }) => ({
 });
 
 /**
- * We require control over _when_ these styles are applied and
- * therefore cannot use @bbc/psammead-visually-hidden-text.
+ * These styles are lifted from @bbc/psammead-visually-hidden-text, which we
+ * cannot use directly as we require control over _when_ they are applied.
```

Comment:
Could we export the style string from visually hidden text and import it here: https://github.com/bbc/psammead/blob/latest/packages/components/psammead-visually-hidden-text/src/index.jsx#L3..L11 ? Just pretty much certain if visually hidden text changes this won't

Reply:
Yeah, we discussed updating `@bbc/psammead-visually-hidden-text` to cater for more use cases in the swarm review. I will raise an issue.